reconciletags/reconciletags.py

```diff
@@ -88,7 +103,11 @@ def reconcile_tags(self, data, dry_run):
                 full_repo = os.path.join(registry, project['repository'])
 
                 for image in project['images']:
-                    full_digest = full_repo + '@sha256:' + image['digest']
+                    digest = self.get_digest_from_prefix(full_repo,
+                                                         image['digest'])
+                    if not digest:
+                        continue
```

Comment:
I think we probably want to blow up and error here instead of continuing. WDYT?

Reply:
I was thinking it could go ahead and tag other images if there were more in the json, but it's actually probably more useful to error